components/Slider/src/MDCSlider.h

```diff
@@ -191,6 +191,14 @@ IB_DESIGNABLE
  Default value is MDCElevationNone.
  */
 @property(nonatomic, assign) MDCShadowElevation thumbElevation UI_APPEARANCE_SELECTOR;
+
+/**
+ The shadow color of the cursor (thumb).
+
+ Default vlaue is black
```

Comment:
Tip: Value

Reply:
Addressed in 17259b9d4826365387da3694fef5d2e2d3fb0542